lib/vfscore/volumes.c

```diff
@@ -0,0 +1,59 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#include <string.h>
+#include <sys/mount.h>
+#include <uk/init.h>
+#include <uk/libparam.h>
+
+#define UK_MAX_VFSCORE_VOLUMES_COUNT			64
```

Comment:
Can we make this available via menu config? Also make it configurable if automount is available?